django_project/ford3/views/qualification_wizard.py

```diff
@@ -235,9 +235,10 @@ def get_context_data(self, form, **kwargs):
         context['provider'] = self.provider
         # make sure logo has been uploaded before set the context
         # otherwise, let it empty
-        if self.qualification.campus.provider.provider_logo:
-            context['provider_logo'] = \
-                self.qualification.campus.provider.provider_logo.url
+        context['provider_logo'] = \
+            self.qualification.campus.provider.provider_logo.url \
```

Comment:
you're repeating yourself four times. 
You should put that logic into a function

```
def get_provider_logo_url(url):
  return (url) ? (url) : ''
```

I haven't tested that code.

Reply:
i can make a function for this if you want.
where to put this function? 
under `views` folder and named it `utils.py`?

Reply:
@boney-bun You should not do it to please me but do it for you and the future developers that will work on this project (it might be me). 
Do you agree that you're repeating yourself? Do you agree that even if your repeating code is quite small, you're adding complexity in four different files?